tt_metal/jit_build/data_format.cpp

```diff
@@ -408,49 +312,35 @@ const DataFormat get_single_pack_src_format(
 }
 
 std::vector<DataFormat> get_pack_src_formats(
-    DataFormat input_formats[NUM_OPERANDS],
-    DataFormat param_formats[NUM_OPERANDS],
-    DataFormat intermed_formats[NUM_OPERANDS],
-    DataFormat output_formats[NUM_OPERANDS],
+    DataFormat data_formats[NUM_CIRCULAR_BUFFERS],
     DataFormat unpack_conditional_dst_format,
     bool fp32_dest_acc_en,
     bool bfp8_pack_precise,
     bool int_fpu_en,
     tt::ARCH arch
 ) {
-    DataFormat pack_output_format = get_pack_data_format(output_formats, intermed_formats);
-
     std::vector<DataFormat> pack_src_formats;
     DataFormat pack_src_format;
-    for (int i = 0; i < NUM_OPERANDS; i++) {
-        pack_src_format = get_single_pack_src_format(input_formats[i], pack_output_format, unpack_conditional_dst_format, fp32_dest_acc_en, bfp8_pack_precise, int_fpu_en, arch);
+    for (int i = 0; i < NUM_CIRCULAR_BUFFERS; i++) {
+        pack_src_format = get_single_pack_src_format(data_formats[i], data_formats[i], unpack_conditional_dst_format, fp32_dest_acc_en, bfp8_pack_precise, int_fpu_en, arch);
```

Comment:
Is this the only usage of get_single_pack_src_format?
Would it make sense to combine the first two parameters into one if they're the same?

Reply:
yes...